packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -557,7 +566,10 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
         target.getAttribute(IS_FOCUSABLE_ATTRIBUTE) === 'true' &&
         target.getAttribute(IS_ENTER_DISABLED_ATTRIBUTE) !== 'true'
       ) {
-        EventGroup.raise(target, 'click', null, true);
+        const event = new Event('MouseEvents');
+
+        event.initEvent('click', true, true);
+        target.dispatchEvent(event);
```

Comment:
Some boilerplate here too. I know EventGroup is bulky, but can we make standalone atomic helpers for these kinds of lines?

Reply:
It's 3 lines to raise an event though.

Reply:
but i will say that it was a pain to remember. `raiseClick` would have been much better

Reply:
let me try to clean that up

Reply:
done!